nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -202,11 +213,11 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         final boolean addFragmentAttrs;
         switch (packagingFormat) {
             case TAR_FORMAT:
```

Comment:
I would. And if there are ways to improve test coverage that would help eliminate the "fear it would break" in the future ;)